wiki/Internet_Relay_Chat/en.md

```diff
@@ -35,47 +32,32 @@ Once you have a client, you need to set up the server settings.
 
 ## Disabling join/part messages
 
-Knowing players who joins or parts is a good thing, however in very crowded channels such as `#osu`, you will constantly receive join/part messages and will not be able to keep track of conversations. Therefore, it's usually preferred that these messages are hidden.
-
-```
-[17:46] * lauripihl (cho@ppy.sh) has left #lobby
-[17:46] * Kastun (cho@ppy.sh) has joined #lobby
-[17:46] * AuReL (cho@ppy.sh) has joined #lobby
-[17:46] * osukd (cho@ppy.sh) has joined #lobby
-[17:46] * BreadTooGood (cho@ppy.sh) has joined #lobby
-[17:46] * keanyew18 (cho@ppy.sh) has joined #lobby
-[17:46] * JaKox (cho@ppy.sh) has joined #lobby
-[17:46] * Kerantor (cho@ppy.sh) has joined #lobby
-```
-
-### Disabling join/part messages in common clients
+Many IRC clients send join and part (leave) messages for when a player has joined/left a channel by defauly. Although useful to some, it can be irritating to others. Users wishing to disable these messages should refer to their prefered client's documentation. A list of common IRC clients and how to disable these messages is described below:
```

Comment:
```suggestion
Many IRC clients send join and part (leave) messages for when a user has joined/left a channel by default. Although useful to some, it can be irritating to others. Users wishing to disable these messages should refer to their preferred client's documentation. A list of common IRC clients and how to disable these messages is described below:
```